tests/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj

```diff
@@ -21,6 +21,7 @@
 
   <ItemGroup>
     <Compile Include="..\fsharp\TestHelpers.fs" Link="TestHelpers.fs" />
+    <Compile Include="ProjectInfoHelpers.fs" Condition="'$(TargetFramework)' == 'net6.0'" />
```

Comment:
Why is target framework condition is necessary?

Reply:
`Ionide.ProjInfo` is a `.net6.0` only thing, https://www.nuget.org/packages/Ionide.ProjInfo#supportedframeworks-body-tab